point_cloud_transport/include/point_cloud_transport/point_cloud_transport.hpp

```diff
@@ -139,11 +139,11 @@ Publisher create_publisher(
 /// \return The advertised publisher
 POINT_CLOUD_TRANSPORT_PUBLIC
 Publisher create_publisher(
-  std::shared_ptr<rclcpp::node_interfaces::NodeInterfaces<
+  rclcpp::node_interfaces::NodeInterfaces<
```

Comment:
Suggestion: Given the repetition of this - you could typedef it to make this more readable. Though, I think also keeping it as-is is useful as a reference for later readers

Reply:
I will also point out that this is changing a public interface, so we usually do a tick-tock deprecation (same goes for all of the public APIs below).

Reply:
These APIs were added on `rolling`, I know I already released the package but I would claim that It was unestable.